index.html

```diff
@@ -10,20 +10,59 @@
   <title>Awesome Books</title>
 </head>
 <body>
+  <header class="header">
```

Comment:
- [ ] [OPTIONAL] Kindly only show one section when the page loads for the first time. Now, the page loads the three sections in the first load of the page after that when I press any link everything works fine.
![image](https://github.com/MargaretTechworld/Awesome-Books/assets/24703171/ce79e479-acb6-4d00-936f-b2cafb2a98fd)